languages/ExplainFieldName/de.ini

```diff
@@ -0,0 +1,4 @@
+;description of fields in explain
+allfields_unstemmed = "Alle Felder"
```

Comment:
I notice that the English strings mention copyField, but the German ones do not. Is this intentional?

Reply:
I changed this. Hopefully this is better understandable now. But as mentioned in the initial description I am not sure about those field explanations. For now these are just three examples. Someone with more experience with the standard VuFind index could/should have a look into it.